### PR TITLE
Newsletter signup form (static)

### PR DESCRIPTION
**Pull request by ltclmbdev**
image 2018-12-19 at 7 06 18 pm

### CODE REVIEW

**Comment by ltclmbdev** [comment]
Can we get `emotion` on this project?

File: src/components/NewsletterSignupForm/images/newsletterLogo.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="46" height="46" viewBox="0 0 46 46">
+  <g fill="none" fill-rule="evenodd">
```
Comment (vojtaholik) [inline src/components/NewsletterSignupForm/images/newsletterLogo.svg]: should be camelCase here? fillRule?

File: src/templates/blog-post.js [hunk 2]
```diff
@@ -38,19 +42,13 @@ class BlogPostTemplate extends React.Component {
...
+          <a href={editUrl} target="_blank" rel="noopener noreferrer">
             Edit on GitHub
           </a>
         </p>
-        <hr
-          style={{
-            marginBottom: rhythm(1),
-          }}
-        />
+        <div style={{ margin: '90px 0 40px 0' }}>
```
Comment (vojtaholik) [inline src/templates/blog-post.js:49]: could rhythm be used here instead of px?

File: src/components/NewsletterSignupForm/index.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+            margin: 0,
+          }}
+        >
+          The Overreacted Newsletter
+        </h3>
+        <p
+          style={{
+            margin: '30px 0 0 0',
+            color: '#191919',
+            fontSize: '24px',
```
Comment (vojtaholik) [inline src/components/NewsletterSignupForm/index.js]: I think we should be using Scale for font sizes.

File: src/components/NewsletterSignupForm/index.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+            }}
+          />
+        </div>
+        <h3
+          style={{
+            textAlign: 'center',
+            fontSize: '1.4427rem',
+            lineHeight: 1.1,
+            color: '#FFA7C4',
+            fontFamily: 'Montserrat',
```
Comment (vojtaholik) [inline src/components/NewsletterSignupForm/index.js]: Should be: fontFamily: 'Montserrat, sans-serif',

File: src/components/NewsletterSignupForm/images/newsletterLogo.svg (hunk 1 above)
Comment (ltclmbdev) [reply src/components/NewsletterSignupForm/images/newsletterLogo.svg]: it's just SVG we import, I don't think we need to change something there

**Pull request merged**